app/controllers/licences.controller.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
```

Comment:
It's become a convention for each module we create that we declare some JSDoc about it at the top. I'd suggest

```suggestion

/**
 * Controller for /licences endpoints
 * @module LicencesController
 */
 
```